maestro-orchestra/src/main/java/maestro/orchestra/yaml/YamlFluentCommand.kt

```diff
@@ -648,9 +672,15 @@ data class YamlFluentCommand(
         return YamlCommandReader.readConfig(runFlowPath).toCommand(runFlowPath).applyConfigurationCommand?.config
     }
 
+    /**
+     * Checks if a path contains JavaScript interpolation patterns like ${...}
+     */
+    private fun containsJsInterpolation(path: String): Boolean {
+        return path.contains("\${")
+    }
```

Comment:
It's a nitpick, but let's not make this specific to paths, since it doesn't need to be, and might be useful and reusable in other instances.

```suggestion
    /**
     * Checks if a string contains JavaScript interpolation patterns, i.e. ${...}
     */
    private fun containsJsInterpolation(string: String): Boolean {
        return string.contains("\${")
    }
```